folksonomy/models.py

```diff
@@ -5,7 +5,7 @@
 from fastapi import FastAPI, status, Response, Depends, Header
 from pydantic import BaseModel, ValidationError, validator
 
-re_barcode = re.compile(r'[0-9]{1,13}')
+re_barcode = re.compile(r'^[0-9]*$')
```

Comment:
We do not want empty barcodes (I know you also test it but, it's better to be precise here).

```suggestion
re_barcode = re.compile(r'^[0-9]+$')
```

Reply:
Some higher limit than 13 seems ok, but we should keep a limit.